lib/tree-view.coffee

```diff
@@ -401,26 +432,26 @@ class TreeView extends View
     selectedEntry = @selectedEntry()
     return unless selectedEntry?
 
-    if directory = $(selectedEntry).closest('.expanded.directory')[0]
+    if directory = selectedEntry.closest('.expanded.directory')
       directory.collapse(isRecursive)
       @selectEntry(directory)
 
   openSelectedEntry: (options={}, expandDirectory=false) ->
     selectedEntry = @selectedEntry()
-    if selectedEntry instanceof DirectoryView
+    if selectedEntry.classList.contains('directory')
```

Comment:
This will break when nothing is selected, because `selectedEntry` is null.

Happened after I hit <kbd>Enter</kbd> with focus on the tree-view by mistake.